group/models.py

```diff
@@ -30,7 +30,7 @@
 
 class Announce(models.Model):
 
-    title = models.CharField(max_length=100, default='')
+    title = models.CharField(max_length=100, default='Required')
```

Comment:
Not sure set default to 'Required' string is good?
I think this column cannot be false?